src/main/resources/swagger/api-docs.yaml

```diff
@@ -9255,6 +9255,9 @@ components:
           description: The date all files were successfully transferred to the workspace bucket if applicable
             in yyyy-MM-ddTHH:mm:ss.SSSZZ format
           format: date-time
+        cloudPlatform:
+          type: string
+          description: The cloud platform of the workspace
```

Comment:
Actually, wondering if we should add the possible values, like we do for the billing project response.

cloudPlatform:
          type: string
          enum:
            - GCP
            - AZURE